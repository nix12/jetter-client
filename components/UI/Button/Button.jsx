import React from 'react';

const button = props => (
  <button onClick={props.clicked} disabled={props.disabled}>
    {props.children}
  </button>
);

export default button;
