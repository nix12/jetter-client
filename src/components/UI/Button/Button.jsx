/* eslint-disable react/button-has-type */
import React from 'react';

const button = props => {
  const { children, clicked, disabled, type } = props;

  return (
    <button onClick={clicked} disabled={disabled} type={type}>
      {children}
    </button>
  );
};

export default button;
