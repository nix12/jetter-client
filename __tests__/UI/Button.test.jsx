import React from 'react';
import { shallow } from 'enzyme';
import Button from '../../components/UI/Button/Button';

describe('button', () => {
  it('should render button component', () => {
    expect(shallow(<Button />)).toMatchSnapshot();
  });
});
