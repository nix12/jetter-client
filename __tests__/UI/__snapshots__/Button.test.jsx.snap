// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button should render button component 1`] = `ShallowWrapper {}`;
